test/04_enumeration/enumeration.py

```diff
@@ -0,0 +1,28 @@
+import unittest
+
+try:
+    import enumeration_pybind11.nested_namespace as py11
+    has_pybind11 = True
+except:
+    has_pybind11 = False
+
+import enumeration_boost_python.nested_namespace as boost
```

Comment:
I think I missed this in the other PR.  Where do the `enumeration_xx_yy` namespaces come from here?

Reply:
That's the name of the binding, which is specified at [here](https://github.com/personalrobotics/chimera/blob/py11_enum/test/04_enumeration/CMakeLists.txt#L3) with suffixes depending on the binding library (`_boost_python` or `_pybind`).